lizmap/modules/admin/zones/project_list.zone.php

```diff
@@ -82,20 +82,17 @@ protected function _prepareTpl()
             }
             $maps[$r] = $lizmapViewItem;
         }
-        // Fixme, to move in VersionTools
         $lizmapTargetVersionInt = jApp::config()->minimumRequiredVersion['lizmapWebClientTargetVersion'];
-        $humanLizmapTargetVersion = substr($lizmapTargetVersionInt, 0, 1);  // Major
-        $humanLizmapTargetVersion .= '.'.ltrim(substr($lizmapTargetVersionInt, 2, 1), ''); // Minor
+        $blockingLizmapVersionInt = ($lizmapTargetVersionInt - 100);
```

Comment:
```suggestion
        // 31500 - 100 → 31400 → 3.14
        $blockingLizmapVersionInt = ($lizmapTargetVersionInt - 100);
```

Just to be clear, adding a comment is welcome sometimes ;-)

Feel free to skip